.travis.yml

```diff
@@ -1,18 +1,55 @@
+sudo: required
```

Comment:
Hmm, this will slow down our Travis runs (which already aren't super fast to begin with)

I'm assuming that's needed to run docker? Is there any way we can only require sudo for some specific parameters, and only run the docker tests for a specific version?